### DISCUSSION TITLE
More different logo

### DISCUSSION THREAD

**Discussion by jcbhmr** [typst-community/org#3] [Ideas]
might I suggest a logo that more clearly differentiates @typst-community from @typst? Specifically: at a small size "t" with "community" written in size 8 font sideways is quite hard to differentiate vs the official @typst logo. The turquoise vs white color difference is good though! I think it might be better with something like "tc" instead of "t"? Just to differentiate https://typst.community or whatever other site in your browser tab favicon from https://typst.app. as-is, the "t"-on-turquoise and "t"-on-white look like "main site" and "docs" versions lol

<img width=60 src="https://avatars.githubusercontent.com/u/67595261?s=200&v=4"> <img width=60 src="https://avatars.githubusercontent.com/u/144882637?s=200&v=4">

<img width=60 src="https://avatars.githubusercontent.com/u/144882637?s=200&v=4"> <img width=60 src="https://i.imgur.com/GjcmkXa.png"> <img width=60 src="https://i.imgur.com/IjWoNJM.png">

ex:
![image](https://github.com/typst-community/.github/assets/61068799/42c8898e-72ba-42b3-beca-cc002aab3f62)

cc @huwaireb since im not sure if you're watching this repo https://github.com/typst-community/.github/watchers

**Comment by huwaireb** [comment]
The original PoC was exactly the second one, however I didn't really like how it turned out at the time so I decided to go with another approach the one you see now:

Differentiate by colors and the community lettering offers visual distinction even if not readable.

Branding needs to be redone sooner or later, imo it's fine if we just keep the one we have now and hold a poll later with the community with more variants as well. Although that is blocked by finishing setting up the typst-community org.

Probably would be before the site is live, which is what matters the most in this case due to favicon of course.

**Comment by huwaireb** [comment]
should we move this to a discussion? @jcbhmr

**Comment by jcbhmr** [comment]
did this ever get decided one way or the other? keep/change?
